src/frequenz/sdk/timeseries/battery_pool/_battery_pool_wrapper.py

```diff
@@ -44,17 +44,17 @@ class BatteryPoolWrapper:
     same set of batteries.
     """
 
-    def __init__(self, battery_pool: BatteryPool, source_id: str | None, priority: int):
+    def __init__(self, battery_pool: BatteryPool, name: str | None, priority: int):
         """Create a BatteryPoolWrapper instance.
 
         Args:
             battery_pool: The battery pool to wrap.
-            source_id: The source ID to use for the requests made by this wrapper.
+            name: The source ID to use for the requests made by this wrapper.
```

Comment:
It's not yet the source id though, only after the unique_id is added, right? Needs docs update :)

Reply:
updated